test/form/datepicker/datepicker.directive.spec.js

```diff
@@ -0,0 +1,46 @@
+describe('Directive: pfDatepicker', function() {
+  var $scope, $compile, $timeout, element, dateInput;
+
+  beforeEach(module('patternfly.form', 'form/datepicker/datepicker.html'));
+
+  beforeEach(inject(function(_$compile_, _$rootScope_) {
+    $compile = _$compile_;
+    $scope = _$rootScope_;
+  }));
+
+  var compileDatepicker = function(markup, $scope) {
+    var el = $compile(markup)($scope);
+    $scope.$apply();
+    return el;
+  };
+
+  it("should set the date picker input", function() {
+    $scope.options = {};
+
+    var datepicker = compileDatepicker('<form><div pf-datepicker options="options" date="date"></div></form>', $scope);
+    var dateInput = angular.element(datepicker).find('input');
```

Comment:
I personally prefer declaring all `var`s together at the top of the function.

If we don't already have this codified I think we should.